default_topic_list:
  - "/carecules_laser/slamtec_raw"
  - "/rplidar_laser_filter/scan_filtered"
  - "/imu/data"
  - "/imu/data_raw"
  - "/imu/data_raw_mag"
  - "/imu/data_flat_world"
  - "/tf"
  - "/tf_static"
  - "/realsense_d435/color/camera_info"
  - "/realsense_d435/color/image_raw"
  - "/realsense_d435/depth/camera_info"
  - "/realsense_d435/depth/image_rect_raw"
  - "/realsense_d435/extrinsics/depth_to_color"
  - "/realsense_t265/accel/imu_info"
  - "/realsense_t265/accel/sample"
  - "/realsense_t265/gyro/imu_info"
  - "/realsense_t265/gyro/sample"
  - "/realsense_t265/odom/sample"
